ansible/roles/common/tasks/main.yml

```diff
@@ -31,6 +31,12 @@
     bin_dir: "/opt/bin"
   when: ansible_distribution == "CoreOS"
 
+# On CentOS with kubernetes installed from packages binaries are in /usr/bin/
+- name: Set the bin directory path for CentOS
+  set_fact:
+    bin_dir: "/usr/bin"
+  when: ansible_distribution == "CentOS" and source_type == "packageManager"
+
```

Comment:
@danehans @eparis I would love to see `/usr/bin` as default `bin_dir`.

I used other approach here because I'm not sure how to properly change `bin_dir` without breaking other configurations which I can't test.

I will rebase this PR on fix of `bin_dir`, if there will be any (I can try to fix it myself, if nobody volunteers).